open-craft/xblock-group-project-v2
# Clearing text in email popups and Ask TA

**Testing instructions:**
1. Configure GPv2. Make sure to have at least one stage with "Project Team" XBlock, and "Ask TA" Project Navigator view
2. Open GPv2 in Apros as a student.
3. Go to  stage with Project Team block.
4. Try emailing group and individual users. Note emails are likely fail to send (unless you have configured smtp relay for devstack box and whatever config is needed in Apros) - this is expected. From GPv2 perspective, sending email is an AJAX request to some URL, and 200 response is considered a success (a message is shown unconditionally though. Unfortunately, Apros responds with 200 even if an email was not sent). So, after sending an email, popup should close (existing behavior) and clear textarea contents (new behavior). Both "email entire team" and individual emails should work like that.
5. Open Ask TA view in PN (button with "TA" on it).
6. Fill in some text, submit. Email won't be sent as well. This time, however, TA view is not closed, so it would be counter-intuitive to remove the message right away. So, textarea is not cleared right away - instead it is cleared when user closes Ask TA view (by opening other view, or using "close view" X in top-right corner)
